CHANGELOG

```diff
@@ -1,4 +1,4 @@
-v2.0.0 Updated to API 2.0.
+v2.0.0 Complete rewrite of the client for API 2.0. Not backwards compatible with the old client or api.
```

Comment:
On second thought, maybe just `Complete rewrite for API 2.0. Not backwards compatible with the old client or api.` Saying client twice seems redundant.